specification/paths/Services-service_id-ServiceContracts.json

```diff
@@ -29,7 +35,7 @@
             "data": {
```

Comment:
Remove `"additionalProperties": false` 2 lines above this one (see previous comment).